src/lib.rs

```diff
@@ -1,4 +1,4 @@
-pub static VERSION: &'static str = "0.82.1";
```

Comment:
Why is this no longer required?

Reply:
I think just better default assumptions in the compiler. It is now able to infer this lifetime automatically.